lib/libsh/libsimbol/libsimbol.sh

```diff
@@ -80,6 +80,11 @@ unset  CDPATH
 export SIMBOL_DEADMAN=${SIMBOL_USER_CACHE}/deadman
 export SIMBOL_IN_COLOR=1
 export SIMBOL_DATE_FORMAT="%x-%X"
+
+declare -gi USER_CPF_INDENT=0
+declare -gi USER_CPF_INDENT_SIZE=2
+declare -g  USER_CPF_INDENT_CHAR=' *'
```

Comment:
So it's just the defaults - note the variable name starts with `USER_`, so it's something I should add to the sample configuration file - I'll do that actually, otherwise the `USER` would have no idea how to set it.

2 vs 4 - that's just personal preference, we can set the default to 4 if you like.

The asterisk is again a user preference - can change that to a space.

Let me update the code and send you another CR.